### PR TITLE
add bodymovin@4.5.3 w/ npm auto-update

### LINKED ISSUE #10329: [Request] Add bodymovin
**Library name:** bodymovin
**Git repository url:** https://github.com/bodymovin/bodymovin
**License(s):** MIT

### PR DESCRIPTION
**Pull request by sufuf3**
close #10329, cc @bodymovin
@kennynaoh please help me review this PR, thank you.

Pull request for issue: #10329 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bodymovin/bodymovin
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/bodymovin
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 105
   - Count of stars: 1745
   - Count of forks: 165
 * NPM download stats (optional):
   - Downloads in the last day: 91
   - Downloads in the last week: 776
   - Downloads in the last month: 2933

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/bodymovin/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "plugin",
+    "export"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/bodymovin/bodymovin/wiki",
+  "author": {
+    "name": "airnan",
+    "email": "hernantorrisi@gmail.com"
+  },
+  "filename": "bodymovin.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/bodymovin/package.json]: Wrong path, didn't you test your auto-update config?

**Comment by sufuf3** [comment]
@PeterDaveHello  I think we don't need bodymovin_light.js, becouse it jest says "get the bodymovin.js file from the build/player/ folder for the latest build" in README.md.
ref: https://github.com/bodymovin/bodymovin/blob/master/README.md#html

**Comment by PeterDaveHello** [comment]
@sufuf3 I don't think that README.md will show you every usage, you need to "think" about it,  the naming is so clear, it gets a minified file, it's just a light version of it, juts like jquery has jquery.slim.js, so why don't we host it?

**Comment by sufuf3** [comment]
@PeterDaveHello  I added the bodymovin_light.js and modified the package.json.
@kennynaoh please help me review this PR, thank you.

**Comment by bodymovin** [comment]
Hi, if it helps, the bodymovin_light.js is a version with less features.
There is no need to add it, but some people might find it helpful.

2017-01-29 14:15 GMT-03:00 Peter Dave Hello <notifications@github.com>:

**Comment by PeterDaveHello** [comment]
@bodymovin Thanks for your explanation, I'll like to let the users decide which one they'll like to use, host both full and light versions can give them the rights to choose :)

**Comment by PeterDaveHello** [comment]
@bodymovin I wonder if we'll only need the files under `build/player` for the front-end? If so, we can drop the `player` directory in the path to make the url shorter since we don't need the others, thanks.

**Comment by bodymovin** [comment]
@PeterDaveHello yes, for front-end, the only files needed are in the "build/player" folder.
Thanks!

**Comment by PeterDaveHello** [comment]
Thanks @bodymovin 

@sufuf3 please update your PR to fit the discussion above.

**Comment by sufuf3** [comment]
@PeterDaveHello I updated it, thank you.
@kennynaoh please help me review this PR, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please help review this PR, thanks.

File: ajax/libs/bodymovin/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "npmName": "bodymovin",
+  "npmFileMap": [
+    {
+      "basePath": "build/play",
```
Comment (pvnr0082t) [inline ajax/libs/bodymovin/package.json]: Wrong basePath.

File: ajax/libs/bodymovin/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      "files": [
+        "**/bodymovin.*"
```
Comment (pvnr0082t) [inline ajax/libs/bodymovin/package.json]: @sufuf3 We can use `"**/*"` here.

Comment (sufuf3) [inline ajax/libs/bodymovin/package.json]: Should I use the following?
"basePath": "build",
"files": [
  "play/*"
]

Comment (bodymovin) [inline ajax/libs/bodymovin/package.json]: The front-end files are at "build/player" instead of "build/play". 
Is that the error?

Comment (sufuf3) [inline ajax/libs/bodymovin/package.json]: @bodymovin  thank you. I didn't notice it.

**Comment by sufuf3** [comment]
pvnr0082t I modified it, please help me review this PR again, thank you.

**Comment by sufuf3** [comment]
@pvnr0082t  sorry, I add these files, , please help me review this PR again, thank you.

**Comment by bodymovin** [comment]
Hi @sufuf3 and @pvnr0082t , 
bodymovin has been renamed to Lottie
could you update the cdn references?
Here's the new url
https://github.com/airbnb/lottie-web/tree/master/build/player

**Comment by sufuf3** [comment]
@bodymovin Thanks
May I ask what package name can be downloaded from npm? (npm install package_name)
Although the README.md said https://github.com/airbnb/lottie-web#html-player-installation, I found the latest version is v4.13.0 from npm.
(If there is no package name from npm, I can use git auto-update. This question just make sure that how to get versions of the lib via auto-update config.)
Many thanks.

@PeterDaveHello May I ask this situation?
The lib name has been renamed from `bodymovin` to `Lottie`
Should I add the `Lottie` library as usually or follow #8904 to redirect old lib to the new one?

Thanks.

**Comment by PeterDaveHello** [comment]
Open a new issue, please.

**Comment by bodymovin** [comment]
@sufuf3 yes, this is the new package name:
https://www.npmjs.com/package/lottie-web
There are 4 files that should replace the previous 4 that were being pulled from the CDN:
https://github.com/airbnb/lottie-web/tree/master/build/player/lottie.js
https://github.com/airbnb/lottie-web/tree/master/build/player/lottie.min.js
https://github.com/airbnb/lottie-web/tree/master/build/player/lottie_light.js
https://github.com/airbnb/lottie-web/tree/master/build/player/lottie_light.min.js

Should I open a new issue myself?

**Comment by sufuf3** [comment]
@bodymovin Thank you.
Many open an issue is the best way.
Would you open an issue?
Thanks.

**Pull request merged**